<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="hostLineEdit">
       <property name="text">
        <string>192.169.1.10</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="portLineEdit">
       <property name="text">
        <string>10001</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="connectToolButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="disconectToolButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Disconect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="4" rowspan="5">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QTableWidget" name="dataTable">
       <column>
        <property name="text">
         <string>2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCustomPlot" name="Plot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Диапазон</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Сек</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox48V">
       <property name="text">
        <string>48 V (motor)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_48V">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_48A">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox8V1">
       <property name="text">
        <string>8V (1)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_81V">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_81A">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox8V2">
       <property name="text">
        <string>8V (2)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_82V">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_82A">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox6V1">
       <property name="text">
        <string>6V (1)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_61V">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_61A">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox6V2">
       <property name="text">
        <string>6V (2)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_62V">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_62A">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>277</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>123</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Time delay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="timeOutSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>0.000100000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="toolButtonStartSending">
     <property name="text">
      <string>Start Sending</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>277</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>122</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Добавить строку</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>connectToolButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>Connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconectToolButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>Disconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonStartSending</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>StartSending()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox48V</sender>
   <signal>clicked(bool)</signal>
   <receiver>Widget</receiver>
   <slot>On48Slot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox8V1</sender>
   <signal>clicked(bool)</signal>
   <receiver>Widget</receiver>
   <slot>On8V1Slot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox8V2</sender>
   <signal>clicked(bool)</signal>
   <receiver>Widget</receiver>
   <slot>On8V2Slot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox6V1</sender>
   <signal>clicked(bool)</signal>
   <receiver>Widget</receiver>
   <slot>On6V1Slot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox6V2</sender>
   <signal>clicked(bool)</signal>
   <receiver>Widget</receiver>
   <slot>On6V2Slot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>AddRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Widget</receiver>
   <slot>SetLenght(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Connect()</slot>
  <slot>Disconnect()</slot>
  <slot>StartSending()</slot>
  <slot>On48Slot(bool)</slot>
  <slot>On8V1Slot(bool)</slot>
  <slot>On8V2Slot(bool)</slot>
  <slot>On6V1Slot(bool)</slot>
  <slot>On6V2Slot(bool)</slot>
  <slot>AddRow()</slot>
  <slot>SetLenght(double)</slot>
 </slots>
</ui>
